src/controller/AutoCommissioner.cpp

```diff
@@ -539,7 +542,7 @@ void AutoCommissioner::SetCASEFailsafeTimerIfNeeded()
         //
         // A false return from ExtendArmFailSafe is fine; we don't want to make
         // the fail-safe shorter here.
-        mCommissioner->ExtendArmFailSafe(mCommissioneeDeviceProxy, CommissioningStage::kFindOperational,
+        mCommissioner->ExtendArmFailSafe(mCommissioneeDeviceProxy, mCommissioner->GetCommissioningStage(),
```

Comment:
This code is completely broken for non-concurrent commissioning, but that's pre-existing. Can you please file an issue on this, assign it to @chrisdecenzo ?

Reply:
https://github.com/project-chip/connectedhomeip/issues/32724, done, thanks